src/assets/YAML/default/BuildAndDeployment/PatchManagement.yaml

```diff
@@ -88,17 +95,27 @@ Build and Deployment:
         iso27001-2017:
         - hardening is missing in ISO 27001
         - 14.2.1
+        d3f:
+        # PlatformHardening
+        - LocalFilePermissions
+        - SystemConfigurationPermissions
+        # ApplicationHardening
+        - DeadCodeElimination
+        - ApplicationConfigurationHardening
```

Comment:
Based on 
>Hardening an application's configuration involves analyzing not only the application but also the environment in which the application is run in for potential vulnerabilities.

I agree. You have done a very detailed mapping.

Reply:
This comes from d3fend ~ for free.